packages/next/next-server/lib/utils.ts

```diff
@@ -191,6 +191,12 @@ export interface NextApiRequest extends IncomingMessage {
   body: any
 
   env: Env
+
+  preview: true | undefined
+  /**
+   * Preview data set on the request, if any
+   * */
+  previewData?: object | string
```

Comment:
```suggestion
  previewData?: any
```

To match `GetStaticPropsContext` & `GetServerSidePropsContext`.